src/components/PlaybookProcess.js

```diff
@@ -82,12 +82,9 @@ class PlaybookProgress extends Component {
       playbooksStarted: [],          // list of playbooks that have started
       playbooksComplete: [],         // list of playbooks that have completed
       playbooksError: [],            // list of playbooks that have errored
-      commit: 'notstarted',          // keep track of the commit
-
-      displayedLogs: List()          // log messages to display in the log viewer
+      displayedLogs: List(),        // log messages to display in the log viewer,
+      commit: this.props.commitStatus ? this.props.commitStatus: undefined
```

Comment:
This should be just ``commit: this.props.commitStatus`` since the parent always sends this field

Reply:
ok, makes sense.